src/main/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticator.java

```diff
@@ -124,7 +139,7 @@ public AuthCredentials extractCredentials(RestRequest request, ThreadContext con
 
         AuthCredentials creds = AccessController.doPrivileged(new PrivilegedAction<AuthCredentials>() {
             @Override
-            public AuthCredentials run() {
+            public AuthCredentials run() throws OpenSearchSecurityException {
```

Comment:
Why this is necessary?

Reply:
This should not be here. I left it here when debugging, will edit it back.